src/components/BookItem.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { deleteBook } from '../redux/books/booksSlice';
+/*eslint-disable*/
+const BookItem = ({
+  item_id, title, author, category,
+}) => {
+    const dispatch = useDispatch();
+
+  return (
+    <>
+      <div className="bookState" key={item_id}>
+        <div>
+          <h3>{title}</h3>
+          <h4>{author}</h4>
+        </div>
+        <button type="button" className="delete"
+        onClick={()=>{
+            dispatch(deleteBook(item_id));
+        }}>Delete</button>
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirements, according to which you should add the `onClick` event on the **Button** component, which will help to remove the specific book from the `redux` store using the reducer. Right now you are using the `button` which is JSX, not a **Component**, which is fine but not fulfilling the requirement. You should re-use the Button.js file for creating the component **OR** Inside the Books.js file you can create one more component with Button and then import it to remove any book using the `click` event to make the things working as it working right now.

### Requirement
![Screenshot from 2023-08-23 21-53-57](https://github.com/MohamedRoshdy2021/Book-store/assets/66476329/c7ad4113-ea5e-4e3f-9794-d95fa9e084dd)